From 9b73f59e234b0426cc7ab44ef244d2439ca02d22 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Sat, 22 Nov 2014 23:56:50 -0500
Subject: [md] dm-thin: remove stale 'trim' message in block comment above pool_message

Message-id: <1416700613-25142-11-git-send-email-snitzer@redhat.com>
Patchwork-id: 100844
O-Subject: [RHEL7.1 PATCH 10/13] dm thin: remove stale 'trim' message in block comment above pool_message
Bugzilla: 1165246
RH-Acked-by: Joe Thornber <thornber@redhat.com>

BZ: 1165246

Upstream linux-dm.git commit 5ec02084f60f1537df850817fb91a16072cba4e7
Author: Mike Snitzer <snitzer@redhat.com>
Date:   Fri Nov 7 15:27:56 2014 -0500

    dm thin: remove stale 'trim' message in block comment above pool_message

    Signed-off-by: Mike Snitzer <snitzer@redhat.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/dm-thin.c b/drivers/md/dm-thin.c
index d3dcd0e..268c688 100644
--- a/drivers/md/dm-thin.c
+++ b/drivers/md/dm-thin.c
@@ -3290,7 +3290,6 @@ static int process_release_metadata_snap_mesg(unsigned argc, char **argv, struct
  *   create_thin <dev_id>
  *   create_snap <dev_id> <origin_id>
  *   delete  <dev_id>
- *   trim  <dev_id> <new_size_in_sectors>
  *   set_transaction_id <current_trans_id> <new_trans_id>
  *   reserve_metadata_snap
  *   release_metadata_snap
-- 
1.7.1